userguide/content/en/docs/adding-content/lookandfeel.md

```diff
@@ -148,6 +148,24 @@ guessSyntax: true
 {{< /tab >}}
 {{< /tabpane >}}
 
+If you are using a Docsy version later than `0.6.0`, the code blocks show a
+"Copy to clipboard" icon in the top right-hand corner. To disable this
+functionality add the `disable_click2copy_chroma` option to your
+configuration file, under `params`:
+
+{{< tabpane persistLang=false >}}
+{{< tab header="Configuration file:" disabled=true />}}
+{{< tab header="hugo.toml" lang="toml" >}}
+disable_click2copy_chroma = true
+{{< /tab >}}
+{{< tab header="hugo.yaml" lang="yaml" >}}
+disable_click2copy_chroma: true
+{{< /tab >}}
+{{< tab header="hugo.json" lang="json" >}}
+"disable_click2copy_chroma": true
+{{< /tab >}}
+{{< /tabpane >}}
```

Comment:
I'm not a fan of the use of these manually coded TOML/YAML/JSON blocks, esp. when the configuration parameter is so simple:

```suggestion
functionality set `params.disable_click2copy_chroma` to `true` in your
configuration file.
```

Reply:
Completely agree. I was mostly following what's already in this doc. Updated.